tools/ci_build/github/linux/test_custom_ops_pytorch_export.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH='/home/neraoof/onnxruntime/tools'
+
+
+pytest /home/neraoof/onnxruntime/tools/test/test_custom_ops_pytorch_exporter.py || exit 1
```

Comment:
hard coded path for your home directory "/home/neraoof"

Reply:
Thanks. I'm doing some tests on pipeline to make sure we handle failures in script and tests correctly. I'm going to mark this as WIP for now until I'm done with trial and error.